supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -286,12 +286,12 @@ The Red Hat Container Catalog no longer exists; it has become part of the Red Ha
 // EAP: General; kept as is
 [[red-hat-customer-portal]]
 ==== image:images/yes.png[yes] Red Hat Customer Portal (noun)
-*Description*: _Red Hat Customer Portal_ is the official name of the customer portal at https://access.redhat.com.
+*Description*: _Red Hat Customer Portal_ is the official name for https://access.redhat.com. Always use _Red Hat Customer Portal_ in the first reference. For subsequent mentions, you can shorten it to _Customer Portal_.
 
-*Use it*: yes
+*Use it*: Yes
```

Comment:
```suggestion
*Use it*: yes
```
All other glossary entries use lowercase "yes". If we want to make this change (to use the initial capital letter), we should make it consistently throughout the glossary.

Reply:
Updated. Thank  you. Made sure there weren't any similar instances to fix in this file.